doc/Interns/letitiakwan/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 5, 2020
```

Comment:
forgot to put in docs.  This should be Date AND Time please

Reply:
Added it in in the newest commit!